### ISSUE TITLE
Remove UI for contribute_suggested_repos

### ISSUE THREAD

**Issue by rkent** [ros-infrastructure/rosindex#396]
When the contributing PR #194 was first landed, it included a small list of repos that would have an expanded contributing section, with a list of the "help wanted" issues for example. That list was supposed to be manually maintained (see (https://github.com/ros-infrastructure/rosindex/pull/194#issuecomment-57231794)

But that list was set initially when the PR was landed, and has never changed:
```yaml
# Repos to list as suggestiong on the "contribute" page
#
# This must be an array of repository names which acts as a whitelist,
# filtering all known repositories by name.
#
contribute_suggested_repos:
  - moveit
  - catkin_virtualenv
  - tiny_tf
```
I'd say that the hope to manually maintain this has not panned out. Instead, I propose that we show the expanded contributing page for any repo that has issues with a 'help wanted' or 'good first issue' tag.

**Comment by tfoote** [comment]
I agree that maintaining this list manually is not working well. It would be great to use the index information to generate/improve content. One of the challenges is that we currently only fetch and render the github tags in the browser to query the issues, which means that we don't have that information available at generation time. And with the current build time I don't think that we want to add querying github as a dependency for the build either. And we don't want this content going stale in our hosting, nor causing churn in the builds. We also can't do a broad set of github queries/searches due to rate limiting. However if we can get it into a single search that might be possible to leverage and render dynamically.

**Comment by rkent** [comment]
"And we don't want this content going stale in our hosting" is sort of humorous, given that the current content (list of packages wanting help) is almost 5 years old.

My current title of this issue is "Remove UI" but you never really said that you support that. Can I just remove it with your support?

The github rate-limiting issues could probably be solved with some sort of caching, but there are a lot of other issues with rosindex I'd like to work on first, so I doubt if that will get done in the foreseeable future. In any case it is beyond scope for this issue.

**Comment by tfoote** [comment]
Cleared in #525
